docs/README.rst

```diff
@@ -41,28 +41,78 @@ Contributing to this repo
 
 Please see `How to contribute <https://github.com/saltstack-formulas/.github/blob/master/CONTRIBUTING.rst>`_ for more details.
 
-Special notes
--------------
+Requisites
+----------
 
 Arvados **requires** a Postgres database for its API server and SSL for communications. If you don't satisfy these two requirements, things
 won't work. It also uses an Nginx server as a redirector but probably almost any other webserver/redirector can be used instead.
 
 We suggest you use the `postgres-formula <https://github.com/saltstack-formulas/postgres-formula/>`_,
 the `nginx-formula <https://github.com/saltstack-formulas/nginx-formula/>`_ and the
-`letsencrypt-formula <https://github.com/saltstack-formulas/letsencrypt-formula/>`_ to install satisfy these dependencies.
-In the FIXME directory there are some example pillar YAMLs to set up these packages as Arvados needs it.
-
-Also, please note that the individual subcomponents `clean` states **won't remove the config**: as the config is common to all the suite
+`letsencrypt-formula <https://github.com/saltstack-formulas/letsencrypt-formula/>`_ to satisfy these dependencies.
+In the **test/salt/pillar/examples/** directory there are example pillar YAMLs to set up these packages, using the mentioned formulas
+as Arvados needs them.
+
+Special notes (a.k.a. how to use this formula)
```

Comment:
A simple "Usage" title might be better

Reply:
Yeah, you're right. I just kept the template's title (special notes) and added to it while writing :yum: